CHANGELOG.rst

```diff
@@ -1,9 +1,35 @@
 .. _changelog:
 
+0.36.0
+------
+
+Renku ``0.36.0`` introduces features and bug fixes to the Knowledge Graph.
+
+User-Facing Changes
+~~~~~~~~~~~~~~~~~~~
+
+**🌟 New Features**
+
+- 🖼️ **Knowledge Graph**: Project Update API can update description, keywords and image
+  (`#1631 <https://github.com/SwissDataScienceCenter/renku-graph/issues/1631>`_).
+
+Internal Changes
+~~~~~~~~~~~~~~~~
+
+**Bug Fixes**
+
+- **Knowledge Graph**: API calls not to remove project access tokens
```

Comment:
I think it makes sense to leave it - since it was causing headaches it's good for us to have a record of it being resolved, but make it explicit about what problem this solved and why

Reply:
What if I leave it (it's actually there) on the KG Release Notes only thinking it's closer to the code causing it? But I'm fine either way.

Reply:
👍